index.js

```diff
@@ -111,12 +108,11 @@ function Launder(options) {
     }
     return s;
 
-
     function fixUrl(href) {
-      if (href.match(/^(((https?|ftp)\:\/\/)|mailto\:|\#|([^\/\.]+)?\/|[^\/\.]+$)/)) {
+      if (href.match(/^(((https?|ftp|mailto):\/\/)|#|([^/.]+)?\/|[^/.]+$)/)) {
         // All good - no change required
         return href;
-      } else if (href.match(/^[^\/\.]+\.[^\/\.]+/)) {
+      } else if (href.match(/^[^/.]+\.[^/.]+/)) {
```

Comment:
Has this been tested? Not 100% sure node will actually tolerate it? Or is it just vscode highlighting that hates it?

Reply:
Yeah, within the brackets they are considered regular characters. I dug in on [this SO](https://stackoverflow.com/questions/47277133/disable-unnecessary-escape-character-no-useless-escape) and confirmed in regex101.com tester.